behat/tests/features/UserManagement/editiingUserDetails.admin.feature

```diff
@@ -0,0 +1,75 @@
+Feature: Editing Deputy details as an admin user
```

Comment:
The filename has an extra `i` in it, should be `editingUserDetails.admin.feature`

Reply:
👀